Routes/Product listing page/product.html

```diff
@@ -0,0 +1,343 @@
+    <link rel="stylesheet" href="/Stylesheets/main.css">
+    <link rel="stylesheet" href="https://maximus-ui.netlify.app/Stylesheets/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+</head>
+
+<body>
+    <div class="page-container">
+        <nav class="main-nav flex-spacearound">
+            <div class="nav-category">
+                <h1 class="brand-name">Shoe-Kart</h1>
+                <a class="category-link men-categ-link" href="#">
+                    <h3> MEN </h3>
+                </a>
+                <a class="category-link women-categ-link" href="#">
+                    <h3> WOMEN </h3>
+                </a>
+                <a class="category-link kids-categ-link" href="#">
+                    <h3> KIDS </h3>
+                </a>
+            </div>
+            <form class="search-input">
+                <input class="search-input" type="search" placeholder="Search...">
+            </form>
+            <div class="nav-icons flex-spacearound">
+                <button class="btn"><a class="link-primary-solid" href="#">
+                        <h4>LOGOUT</h4>
+                    </a></button>
+                <div class="user-icon badge text-center">
+                    <i class="fa fa-user-circle"></i>
+                    <h4>Profile</h4>
+                </div>
+                <div class="wishlist-icon badge text-center">
+                    <i class="fa fa-heart"></i>
+                    <span id="wishlist-count">9</span>
+                    <h4>Wishlist</h4>
+                </div>
+                <div class="cart-icon badge text-center">
+                    <i class="fa fa-shopping-cart"></i>
+                    <span id="cart-count">9</span>
+                    <h4>Cart</h4>
+                </div>
+            </div>
+        </nav>
+        <!-- navbar ends -->
+    <div class="pdt-page-content"> 
+     <div class="side-bar">
+         <div class="filter flex-spacebetween margin-bottom">
+             <h3>Filters</h3>
+             <a href="#" class="auth-link">Clear</a>
+         </div>
+         <div class="price-filter margin-bottom">
+             <h3 class="little-margin-bottom">Price</h3>
+             <input type="range" min="0" max="100" value="50" step="50"class="slider-input" />
+                    <div class="price-range flex-spacebetween">
+                        <p>1k</p>
+                        <p>5k</p>
+                        <p>10k+</p>
+                    </div>
+        </div>
+        <div class="category-filter margin-bottom">
+            <h3 class="little-margin-bottom">Category</h3>
+            <div class="checkbox-ip">
+                <input id="men-cloathing" name="men" type="checkbox" checked>
+                <label for="men-cloathing"> Men Clothing</label>
```

Comment:
there's a typo

Reply:
thanks for pointing,
I have addressed this